sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClientJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,353 @@
+                                System.out.printf("%s ", recognizedTableCell.getText())));
+                    })
+            ));
+        // END: com.azure.ai.formrecognizer.FormRecognizerAsyncClient.beginRecognizeContent#Flux-long-FormContentType
+    }
+
+    /**
+     * Code snippet for {@link FormRecognizerAsyncClient#beginRecognizeContent} with options
+     *
+     * @throws IOException Exception thrown when there is an error in reading all the bytes from the File.
+     */
+    public void beginRecognizeContentWithOptions() throws IOException {
+        // BEGIN: com.azure.ai.formrecognizer.FormRecognizerAsyncClient.beginRecognizeContent#Flux-FormContentType-long-Duration
+        File sourceFile = new File("{file_source_url}");
+        Flux<ByteBuffer> buffer = Utility.convertStreamToByteBuffer(
+            new ByteArrayInputStream(Files.readAllBytes(sourceFile.toPath())));
+
+        formRecognizerAsyncClient.beginRecognizeContent(buffer, FormContentType.APPLICATION_PDF, sourceFile.length(),
+            Duration.ofSeconds(5)).subscribe(recognizePollingOperation ->
+                recognizePollingOperation.getFinalResult().subscribe(layoutPageResults ->
+                    layoutPageResults.forEach(recognizedForm -> {
+                        System.out.printf("Page Angle: %s%n", recognizedForm.getTextAngle());
+                        System.out.printf("Page Dimension unit: %s%n", recognizedForm.getUnit());
+                        // Table information
+                        System.out.println("Recognized Tables: ");
+                        recognizedForm.getTables().forEach(formTable ->
+                            formTable.getCells().forEach(recognizedTableCell ->
+                                System.out.printf("%s ", recognizedTableCell.getText())));
+                    })
+                ));
+        // END: com.azure.ai.formrecognizer.FormRecognizerAsyncClient.beginRecognizeContent#Flux-FormContentType-long-Duration
+    }
+
+    // Recognize Receipts
+    /**
+     * Code snippet for {@link FormRecognizerAsyncClient#beginRecognizeReceiptsFromUrl}
+     */
+    public void beginRecognizeReceiptsFromUrl() {
+        // BEGIN: com.azure.ai.formrecognizer.FormRecognizerAsyncClient.beginRecognizeReceiptsFromUrl#string
+        String receiptUrl = "{file_source_url}";
+        formRecognizerAsyncClient.beginRecognizeReceiptsFromUrl(receiptUrl).subscribe(recognizePollingOperation -> {
+            // if training polling operation completed, retrieve the final result.
+            recognizePollingOperation.getFinalResult().subscribe(recognizedReceipts ->
+                recognizedReceipts.forEach(recognizedReceipt -> {
+                    USReceipt usReceipt = ReceiptExtensions.asUSReceipt(recognizedReceipt);
+                    System.out.printf("Page Number: %s%n", usReceipt.getMerchantName().getPageNumber());
+                    System.out.printf("Merchant Name: %s%n", usReceipt.getMerchantName().getFieldValue());
```

Comment:
Need to print merchant address and tax and phone number just to show various us receipt items.

Reply:
we need only the values , not so much name.